### ISSUE TITLE
Objects passed to self.log_<level> do not get rendered

### ISSUE THREAD

**Issue by Mailstorm-ctrl** [netbox-community/netbox#16453]
### Deployment Type

Self-hosted

### NetBox Version

v4.0.5

### Python Version

3.12

### Steps to Reproduce

1. Create a custom script with any kind of log message
2. Pass an object using `obj=` or as the second argument
3. Run the script

Example script:
```python
class examplev2(Script):
        region = ObjectVar(
                model=Region
                )
        def run(self, data, commit):
                def get_child_regions(region, ids=None):
                        if ids is None:
                                ids = []
                        if len(region.children.all()) == 0:
                                self.log_debug(f"No children in {region.name}")
                                ids.append(region.id)
                                return ids
                        for child in region.children.all():
                                self.log_debug(f"Children in {region.name}")
                                ids.append(child.id)
                                get_child_regions(child, ids)
                        return ids
                ids = get_child_regions(data['region'])
                devices = [device.name for device in Device.objects.all() if device.site.region.id in ids]
                self.log_info(f"Devices: {devices}",data['region'])
```

### Expected Behavior

The passed object to be available in the output. In older versions there was a column that had a link to the objects page.

### Observed Behavior

No column or mention of the passed object.
![image](https://github.com/netbox-community/netbox/assets/53045430/a780d1b9-e652-4f94-a290-540bb4e4683a)

**Comment by Mailstorm-ctrl** [comment]
I mentioned this in https://github.com/netbox-community/netbox/issues/15983 but figured it should have its own report

**Comment by jeremystretch** [comment]
Sorry for the delay in responding. I think there's some overlap between this and #16307, which has a PR open. I'm going to mark this as blocked for now, then re-evaluate once the fix for #16307 has been merged.

**Comment by Mailstorm-ctrl** [comment]
This seems to be fixed now. Though I now need to refresh the page to view the results after a script completes. I'll open a separate issue for that
